portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsWebContent.testcase

```diff
@@ -139,6 +139,162 @@ definition {
 			locator1 = "WC#ENTRY_LIST_TITLE");
 	}
 
+	@description = "LPS-136380 - Add localization for translated titles and content."
+	@priority = 4
+	test CanReviewChangeForWebContentWithLocalization {
+		task ("Create new Publication and new structure with localization") {
+			JSONPublications.addPublication(publicationName = "Publication Name");
+
+			JSONPublications.selectPublication(publicationName = "Publication Name");
+
+			WebContentStructures.openWebContentStructuresAdmin(siteURLKey = "Guest");
+
+			WebContentStructures.addCP(structureName = "WC Structure Name");
+
+			DataEngine.addField(
+				fieldFieldLabel = "Text",
+				fieldName = "Text");
+
+			WebContentStructures.addLocalization(
+				locale = "de-DE",
+				localization1Name = "WC Struktur Name");
+
+			WebContentStructures.saveCP(structureName = "WC Struktur Name");
+		}
+
+		task ("Create new Web Content with structure and new template with localization") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "Guest");
+
+			WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Name");
+
+			WebContent.addWithStructureCP(webContentTitle = "WC Title");
+
+			PortletEntry.changeLocale(locale = "de-DE");
+
+			WebContent.addWithStructureCP(webContentTitle = "WC Titel");
+
+			PortletEntry.publish();
+
+			WebContentNavigator.openWebContentTemplatesAdmin(siteURLKey = "Guest");
+
+			WebContentTemplates.addCP(templateName = "WC Template Name");
```

Comment:
@danielCamara02, thanks for working on it, we'd better create the template with the structure that you created before, and the create order should be structure, template then web content, can you update it?